src/GraphQL.Tests/Bugs/BubbleUpTheNullToNextNullable.cs

```diff
@@ -238,7 +238,7 @@ private void AssertResult(string query, string expected, Data data, IReadOnlyLis
         {
             actualErrors.Count.ShouldBe(errors.Count);
 
-            for (var i = 0; i < errors.Count; i++)
+            for (int i = 0; i < errors.Count; i++)
```

Comment:
This also I wouldn’t implement this suggestion

Reply:
https://github.com/graphql-dotnet/graphql-dotnet/blob/ac4f9c94e79163036fad38510b2829de4e800f1b/.editorconfig#L87

Reply:
Note, that only test project was changed. Others have already conform this rule.

Reply:
Ok